components/admin_console/feature_discovery/features/images/announcement_banner_svg.tsx

```diff
@@ -5,351 +5,545 @@ import React from 'react';
 
 const AnnouncementBannerSVG = () => (
     <svg
```

Comment:
since these are no longer just simple svg images, it makes more sense to group them in the folder components > common > svg_images_components . Also, it would be good if you can make the width and height properties of the svg component properties with default values, so in case that we potentially need this svg component in any other place with another design size, we can just easily reuse it.

Reply:
Agreed with the above, however, given that these are feature discovery specific, it makes sense to keep under the `/feature_discovery` path.

Reply:
I opted to add the props and keep the path same since for now they are feature discovery specific.